tasks/browserstack.js

```diff
@@ -122,6 +122,7 @@ const runr = function() {
  * Text: "Failure"
  * @see http://patorjk.com/software/taag/#p=display&f=Bloody&t=Failure
  */
+// eslint-disable-next-line no-redeclare
```

Comment:
I don't get why it should complain about redeclare, where it's being declared before?

Reply:
If we remove this assign, eslint will start to complain `error : Do not use 'new' for side effects no-new`
Which rule you want to disable? :)
I chose this one to introduce a minimal changes.

Reply:
You are right. Probably there was some mix-up with plugins (I updated not all of them in one go) and I had a false-positive on this line. Now it do not complain, so I will remove this.